test/__snapshots__/cssMinimizerOptions-option.test.js.snap.webpack4

```diff
@@ -11,12 +10,15 @@ exports[`when applied with "minimizerOptions" option matches snapshot for "disca
   /*! this is a comment with exclamation mark */color:red}"
 `;
 
-exports[`when applied with "minimizerOptions" option matches snapshot for "discardComments" option (enable, with "removeAll" option): entry.css 1`] = `"body{color:red}"`;
+exports[`when applied with "minimizerOptions" option matches snapshot for "discardComments" option (enable, with "removeAll" option): entry.css 1`] = `
+"body{
+  /*! this is a comment with exclamation mark */color:red}"
+`;
 
-exports[`when applied with "minimizerOptions" option matches snapshot for "discardEmpty" option (disable): entry.css 1`] = `"body{color:red}a{}"`;
+exports[`when applied with "minimizerOptions" option matches snapshot for "discardEmpty" option (disable): entry.css 1`] = `"body{color:red}"`;
```

Comment:
@ludofischer Regression... maybe on `cssnano` side? As you can see `discardEmpty` is disabled here

Reply:
Passing options to the plugins fails everywhere. This usage is not covered in any tests in the cssnano repository. Even if the options reach the plugins, I don't understand how just passing `pluginName: false` would turn them off.

Reply:
I will look at this in near future

Reply:
I have found passing the `minimzerOptions` to `cssnano()` instead of `process()` fixes it.

Reply:
We should fix it on cssnano side or look on workarounds, we should not break API, because many tools uses `process`

Reply:
At the moment the API is broken already because you cannot do`cssnano.process()` anymore, you must call `cssnano` like a function:

```
cssnano().process()
```
The reason is that PostCSS 8  does not add a `process()` method to plugins any more, so we already added a workaround  and we return a `postcss` instance, which has a `process()` method, but you need to instantiate it to call it. The API we ended with in 5 is

```
cssnano(csssnano-specific options).process(postcssoptions)
```